### PR TITLE
Adds gradle configuration for SimpleDigital watch face

### PR DESCRIPTION
**Pull request by garanj**
Adds gradle configuration for `SimpleDigital` watch face example.

Once added, other watch faces to follow, plus updated documentation.

### CODE REVIEW

File: WatchFaceFormat/SimpleDigital/watchface/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        applicationId = "com.example.simpledigital"
+        minSdk = 33
+        targetSdk = 33
+        versionCode = 1
+        versionName = "1.0.0"
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = true
```
Comment (yschimke) [inline WatchFaceFormat/SimpleDigital/watchface/build.gradle.kts:34]: Do you want to add `isShrinkResources = false`. I'm assuming we should disable? Probably with a comment.

Comment (yschimke) [inline WatchFaceFormat/SimpleDigital/watchface/build.gradle.kts:34]: Does this build ok without a signing key?

Comment (garanj) [reply WatchFaceFormat/SimpleDigital/watchface/build.gradle.kts:35]: It does build, though it doesn't deploy. I've added a comment to show that the developer should add a signingConfig.

Comment (garanj) [reply WatchFaceFormat/SimpleDigital/watchface/build.gradle.kts:35]: I've sent a question to the watch face team. Will update on their response.

Comment (yschimke) [reply WatchFaceFormat/SimpleDigital/watchface/build.gradle.kts:35]: I assume the build system won't understand that WFF references resources.

**Pull request merged**